File: src/codes/Users_Manual/gain/bessel_0.py
~from pyautd3 import Bessel, BesselOption, Intensity, Phase, rad
~
~x = 0.0
~y = 0.0
~z = 0.0
~nx = 1.0
~ny = 0.0
~nz = 0.0
~theta = 0.0
Bessel(
    apex=[x, y, z],
    direction=[nx, ny, nz],
    theta=theta * rad,
    option=BesselOption(
        intensity=Intensity.MAX,
        phase_offset=Phase.ZERO,
    ),
)

File: src/codes/Users_Manual/gain/bessel_0.rs
# use autd3::prelude::*;
# fn main() {
# let x = 0.;
# let y = 0.;
# let z = 0.;
# let nx = 0.;
# let ny = 0.;
# let nz = 0.;
# let theta = 0. * rad;
# let _ = 
Bessel {
    apex: Point3::new(x, y, z),
    dir: UnitVector3::new_normalize(Vector3::new(nx, ny, nz)),
    theta,
    option: BesselOption {
        intensity: Intensity::MAX,
        phase_offset: Phase::ZERO,
    },
};
# }